fmcadc5/fmcadc5.c

```diff
@@ -198,6 +198,7 @@ int main(void)
 	i5g_init_param.ad9625_cs_0 = 1;
 	i5g_init_param.ad9625_cs_1 = 2;
 	i5g_init_param.regs = XPAR_AXI_FMCADC5_SYNC_BASEADDR;
+	i5g_init_param.sysref_delay = 0;
```

Comment:
this is good;
another way to do it, is to do a `memset(&i5g_init_param, 0, sizeof(i5g_init_param))` right before these assignments ; 
this typically makes sure that all memory is 0, except for the parameters which are specifically initialized;

Reply:
ack: disregard my comment :)